packages/frontend-2/lib/projects/composables/projectManagement.ts

```diff
@@ -551,11 +552,30 @@ export function useMoveProjectToWorkspace() {
   const { triggerNotification } = useGlobalToast()
   const mixpanel = useMixpanel()
 
-  return async (projectId: string, workspaceId: string, workspaceName: string) => {
+  return async (
+    projectId: string,
+    workspaceId: string,
+    workspaceName: string,
+    eventSource?: string
+  ) => {
     const { data, errors } = await apollo
       .mutate({
         mutation: useMoveProjectToWorkspaceMutation,
-        variables: { projectId, workspaceId }
+        variables: { projectId, workspaceId },
+        update(cache) {
+          if (!workspaceId) return
```

Comment:
You should check if the update actually succeeded. Check the mutation response result (passed in as a param to update()).